scalatest/src/main/scala/org/scalatest/tools/ScalaTestFramework.scala

```diff
@@ -117,7 +118,7 @@ class ScalaTestFramework extends SbtFramework {
         def annotationName = "org.scalatest.WrapWith"
         def isModule = false
       }
-    )
+    ).asInstanceOf[Array[Fingerprint | Null]]
```

Comment:
**Overridden java array field**
A Scala field overrides some Java array field. We need to use `Array[T | Null]` instead of `Array[T]` for the overriding to work.